governance/votes/Trial Executive vote - Aug 2, 2023.md

```diff
@@ -0,0 +1,142 @@
+
+See this forum [post](https://forum.makerdao.com/t/clydesdale-quarterly-return-of-surplus-fund/21291) for additional context.
+
+## Review
+
+Community debate on these topics can be found on the MakerDAO [Governance forum](https://forum.makerdao.com/). Please review any linked threads to inform your position before voting.
+
+---
+
+## Resources
+
+Additional information about the Governance process can be found in the [Maker Operational Manual](https://manual.makerdao.com).
+
+To add current and upcoming votes to your calendar, please see the [MakerDAO Governance Calendar](https://manual.makerdao.com/makerdao/calendars/governance-calendar).
+
+# Meta-Notes from DAO Masons
+
+- In New Silver, we couldn't really tell what the 'DOC variable' was about or exactly what's being updated. It's in the spreadsheet so it's clearly important, but the originating [post](https://forum.makerdao.com/t/rwa-002-new-silver-restructuring-risk-and-legal-assessment/21417/15) doesn't give much context. Tried to walk the 'clear enough but still vague so it's not technically wrong' line but would have liked to give more context here. Suspect there is some documentation on this that would make it clear, but we didn't find it. We'd be sure to ask about this.
+
+- We weren't sure about our read on the 'MakerDAO Resolution' part of the Clydesdale/JAT1 fund liquidation. Is a 'Resolution' something like a signal vote or attestation that lets Maker sign a document? That's the assumption we went with, and we assumed no actual transaction was happening other than a signal, even though the Clydesdale guy was saying he wanted to liquidate funds. We're surmising those assets are outside of MakerDAO.
+
+- In the spreadsheet the "Spark Proxy Spell" was a separate entry at the bottom from the debt ceiling increase. We chose to combine those because it made sense to do Spark changes together, but were still wondering at the reasoning of having it separate. It was two polls, and that makes sense. But the "spark proxy spell" for some variables but not the debt ceiling was hard to follow without context. Also the contract hash was provided in the spreadsheet but we couldn't find that reference in the forum, so we wonder where it came from.
+
+- We weren't sure who to include for $exec_publishers and $exec_crafters at the top. We saw that AmusingAxl, Sidestream, Arby were listed as 'technical point' and 'reviewers' in the spreadsheet so we went with them. In real life, I'd want to check that assumption.
+
+- This brings up the next question of how the spreadsheet gets filled in, and if we'd be responsible for coming up with those values that appear there. I believe you said it is collaboratively created, but since the spreadsheet provides a lot of our context to write executives, we wonder what context is required to create the spreadsheet.
```

Comment:
It's a collaboration between the technical teams and the governance teams. And honestly just kind of all the facilitators for the various scope.